server/monitor.go

```diff
@@ -1281,6 +1282,13 @@ func (s *Server) createVarz(pcpu float64, rss int64) *Varz {
 		}
 		varz.LeafNode.Remotes = rlna
 	}
+	if l := len(opts.TrustedOperators); l > 0 {
```

Comment:
We can't change TrustedOperators signature, this is an option that exists in public release. Need to rethink your approach.

Reply:
fixed